### PR TITLE
TEAM D WIKI Entry 2020 Fall

### PR DESCRIPTION
**Pull request by ZhekaiJin**
Add the proposed Python Libraries for Reinforcement Learning for Team D
Author: Alvin Huang, Rohan Pandya, Mayank Singal, Karthik Arcot, Scott Jin

### CODE REVIEW

**Comment by ZhekaiJin** [comment]
Corrected wrong path hiding existing page in the existing repo

File: _data/navigation.yml [hunk 1]
```diff
@@ -98,10 +98,12 @@ wiki:
       - title: Pure Pursuit Controller for Skid Steering
         url: /wiki/actuation/Pure-Pursuit-Controller-for-Skid-Steering-Robot.md
   - title: Machine Learning
-    url: /wiki/deep-learning/custom-semantic-data
+    url: /wiki/machine-learning/
     children:
       - title: Custom data-set for segmentation
-        url: /wiki/actuation/motor-controller-feedback/
+        url: wiki/machine-learning/custom-semantic-data/
```
Comment (smbryan) [inline _data/navigation.yml:104]: Thank you for correcting this.

File: wiki/machine-learning/python-libraries-for-reinforcement-learning.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
...ipedia.org/wiki/Reinforcement_learning)is a machine learning approach for teaching agents how to solve tasks by trial and error. More specifically, RL is mostly concerned with how software agents should take actions in an environment in order to maximize its cumulative reward. The application of RL, as it seeks a solution to balance exploration and exploitation, ranges from Resource Management, Traffic Light Control, Recommendation, and Advertising, to Robotics. A survey on RL with Robotics could be found [here](https://www.ias.informatik.tu-darmstadt.de/uploads/Publications/Kober_IJRR_2013.pdf). The successes of deep learning and reinforcement learning area in recent years have led many researchers to develop methods to control robots using RL with the motivation to automate the process of designing sensing, planning, and control algorithms by letting the robot learn them autonomously. This post gives a brief introduction to a few popular RL libraries in the Robotics context, from beginning to immediate, to advanced level users. At last, we will provide general tips for a more in-depth study on RL topics and link to a concrete example of using RL to formulate a self-driving agent.
```
Comment (smbryan) [inline wiki/machine-learning/python-libraries-for-reinforcement-learning.md:5]: The links in the header here won't display properly when the summaries get generated. 

From the template: `The first 100 words are used as an excerpt on the Wiki's Index. No images, HTML, or special formating should be used in this section as it won't be displayed properly.`

They should be moved to References or below the first subheading (i.e. "Spinning Up")

**Review by smbryan — CHANGES_REQUESTED** [review]
Solid submission. The first paragraph needs some attention and we're ready to merge. Sadly, this is a limitation of Jekyll.

File: wiki/machine-learning/python-libraries-for-reinforcement-learning.md (hunk 2 above)
Comment (ZhekaiJin) [reply wiki/machine-learning/python-libraries-for-reinforcement-learning.md]: Thanks and I moved them to See Also links!

**Comment by smbryan** [comment]
Merged. Enjoy your break!

**Pull request merged**